modules/fluid_properties/test/tests/saline/tests

```diff
@@ -0,0 +1,11 @@
+[Tests]
+  [test]
+    type = CSVDiff
+    input = 'test.i'
+    csvdiff = 'test_out.csv'
+    rel_err = 1e-8
+    allow_test_objects = true
+    threading = '!pthreads'
+    required_objects = 'SalineFluidProperties'
```

Comment:
This needs to be changed
```suggestion
    required_submodule = 'contrib/saline'
```

Because the object now always exists (albeit in a defunct state if the module is not available).

Reply:
I will include this with the other changes.